openmetadata-ui/src/main/resources/ui/src/locale/languages/zh-cn.json

```diff
@@ -178,16 +178,16 @@
     "column-profile": "列分析",
     "comment": "评论",
     "comment-lowercase": "评论",
-    "complete": "Complete",
+    "complete": "完成",
     "completed": "已完成",
     "completed-entity": "已完成{{entity}}",
-    "compute-row-count": "Compute Row Count",
+    "compute-row-count": "计算行数",
     "condition": "条件",
     "config": "配置",
     "configuration": "配置",
-    "configure": "Configure",
+    "configure": "配置",
     "configure-a-service": "配置服务",
-    "configure-dbt-model": "配置 dbt 模型",
+    "configure-dbt-model": "配置 DBT 模型",
```

Comment:
this has to be in the lowercase according to the dbt brand guidelines

Reply:
I thought it was a capitalized acronym, but I'll change it back. 🤣